src/main/java/org/opentripplanner/updater/GraphWriterRunnable.java

```diff
@@ -18,5 +20,5 @@
     /**
      * This function is executed to modify the graph.
      */
-    public void run(Graph graph);
+    public void run(Graph graph) throws ParseException;
```

Comment:
Is this required? Don't see any changed implementation(?)

Reply:
Probably just an artifact of an experiment.